datafusion/common/src/utils.rs

```diff
@@ -464,4 +483,47 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_quote_identifier() -> Result<()> {
```

Comment:
This test verifies that identifiers are properly quoted (and that when parsed, quoted identifiers are the same)